packages/react-internal/src/components/Fabric/Fabric.test.tsx

```diff
@@ -1,84 +1,78 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import * as renderer from 'react-test-renderer';
-import { mount } from 'enzyme';
+import { create } from '@uifabric/utilities/lib/test';
+import { Customizer } from '@uifabric/utilities';
 import { Fabric } from './Fabric';
-// import { Checkbox } from '../../Checkbox';
-// import { Customizer } from '../../Utilities';
-// import { createTheme } from '../../Styling';
+import { Checkbox } from '@fluentui/react/lib/Checkbox';
```

Comment:
@dzearing This component is not present in react-internal. What component within react-internal would you recommend using instead?

Reply:
For now let's just comment out this import and the parts of the test that use it, unless someone has a suggestion right away for another component to use.